components/services/bat_ledger/bat_ledger_client_mojo_proxy.cc

```diff
@@ -569,6 +569,10 @@ void OnGetActivityInfoList(const ledger::PublisherInfoListCallback& callback,
   callback(std::move(publisher_info_list), next_record);
 }
 
+void BatLedgerClientMojoProxy::OnGrantViaSafetynetCheck(const std::string& promotion_id, const std::string& nonce) {
```

Comment:
`SafetyNet` is an Android API. iOS has a `DeviceCheck` API so we should decouple to platform specific classes, using naming such as `FraudCheck` so the check is platform agnostic

Reply:
Issue created for that https://github.com/brave/brave-browser/issues/6200